src/main/java/org/sagebionetworks/web/client/presenter/ACTDataAccessSubmissionsPresenter.java

```diff
@@ -204,7 +213,7 @@ public void loadMore() {
 		loadMoreContainer.setIsProcessing(true);
 		synAlert.clear();
 		// ask for data access submissions once call is available, and create a widget to render.
-		jsClient.getDataAccessSubmissions(actAccessRequirementId, nextPageToken, stateFilter, SubmissionOrder.CREATED_ON, isSortedAsc, new AsyncCallback<SubmissionPage>() {
+		jsClient.getDataAccessSubmissions(actAccessRequirementId, accessorId, nextPageToken, stateFilter, SubmissionOrder.CREATED_ON, isSortedAsc, new AsyncCallback<SubmissionPage>() {
```

Comment:
Note that date range values are not used in the call (and were hidden in the UI).  So I removed them.  Added accessorId filter support.